### ISSUE TITLE
FFMpegFrameGrabber fails to identify proper resolution with multiple SPS/PPS entries in H264 encoded stream

### ISSUE THREAD

**Issue by robertoandrade** [bytedeco/javacv#77]
I have a video that changes orientation during playback which causes the insertion of additional SPS/PPS NALUs specifying the new stream dimension but even though the underlying library detects the change and logs it accordingly (like `ffmpeg` and `ffplay` do) the grabber instance I have will still spit me frames with the previous dimensions. Is there a way to reset it upon detection of the change?

I've trying setting `setImageWidth/Height` before starting the stream but couldn't find a way to do it mid way through playback.

Initial log feedback:

```
Input #0, h264, from 'src/test/resources/videos/orientation-change.h264':
  Duration: N/A, bitrate: N/A
    Stream #0:0: Video: h264 (High), yuvj420p(pc), 364x648, 25 fps, 25 tbr, 1200k tbn, 50 tbc
```

When the extra SPS/PPS entries are found:

```
[h264 @ 0x7f97b4a6c000] Reinit context to 1152x656, pix_fmt: yuvj420p
```

**Comment by robertoandrade** [comment]
From what I just checked when upgrading to the latest javacv and ffmpeg preset I noticed the grabber source changed a bit so this solution doesn't work on the latest and from my tests it's still an issue on the latest. Is there any other recommended way to deal with the orientation change and reinitializing the internal state?

**Comment by saudet** [comment]
@robertoandrade We can still do exactly the same thing. I'm just waiting for someone to produce a proper patch...

**Comment by robertoandrade** [comment]
Ok, will try to do that today... updating my logic and testing it right now.

**Comment by saudet** [comment]
Any updates?

**Comment by saudet** [comment]
I'm pretty sure pull #769 fixes this. Please let me know if you still have issues with this! Thanks

**Comment by saudet** [comment]
Thanks to @mifritscher, support is now included in version 1.4! Enjoy
